ggshield/sca/client.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from typing import Dict, List, Optional, Union, cast
+
+import marshmallow_dataclass
+from pygitguardian.client import GGClient, is_ok, load_detail
+from pygitguardian.models import Base, BaseSchema, Detail, FromDictMixin
+
+
+@dataclass
+class ComputeSCAFilesResult(Base, FromDictMixin):
+    sca_files: List[str]
+    potential_siblings: List[str]
+
+
+ComputeSCAFilesResult.SCHEMA = cast(
+    BaseSchema,
+    marshmallow_dataclass.class_schema(ComputeSCAFilesResult, base_schema=BaseSchema)(),
+)
+
+
+class SCAClient:
+    def __init__(self, client: GGClient):
+        self._client = client
+
+    def compute_sca_files(
+        self,
+        touched_files: List[str],
+        extra_headers: Optional[Dict[str, str]] = None,
+    ) -> Union[Detail, ComputeSCAFilesResult]:
+        response = self._client.get(
+            endpoint="sca/compute_sca_files/",
+            params={"touched_files": ",".join(touched_files)},
```

Comment:
A bit out of scope, but I feel this endpoint should be a POST with a `{ "touched_files": [...}}` payload instead of a GET joining all files in a single parameter.